crates/primitives/Cargo.toml

```diff
@@ -90,6 +90,7 @@ pprof = { version = "0.12", features = ["flamegraph", "frame-pointer", "criterio
 [features]
 default = []
```

Comment:
should we make them default?

Reply:
sort of indifferent to what it should be tbh, but let's make it default for now